### PR TITLE
mediahuman apps: updated min system requirements 

### PR DESCRIPTION
**Pull request by kvorobyov**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/y/youtube-to-mp3.rb [hunk 1]
```diff
@@ -15,6 +15,8 @@
     regex(/>\s*Version:.*?(\d+(?:\.\d+)+)/i)
   end
 
+  depends_on macos: "> :sierra"
```
Comment (khipp) [inline Casks/y/youtube-to-mp3.rb:18]:
```suggestion
  depends_on macos: ">= :sierra"
```

File: Casks/m/mediahuman-youtube-downloader.rb [hunk 2]
```diff
@@ -14,6 +14,8 @@
     regex(/Version:.*?(\d+(?:\.\d+)+)/i)
   end
 
+  depends_on macos: "> :sierra"
```
Comment (khipp) [inline Casks/m/mediahuman-youtube-downloader.rb:17]:
```suggestion
  depends_on macos: ">= :sierra"
```

File: Casks/y/youtube-to-mp3.rb (hunk 1 above)
Comment (kvorobyov) [reply Casks/y/youtube-to-mp3.rb:18]: actually sierra is not supported, so it's better to 
`depends_on macos: ">= : high_sierra"` 
right?

Comment (khipp) [reply Casks/y/youtube-to-mp3.rb:18]: Based on the homepage, macOS Sierra is compatible with both apps:

image

Comment (kvorobyov) [reply Casks/y/youtube-to-mp3.rb:18]: There are special links for macOS Sierra (and as far as I know, they will be dropped soon), but I couldn't figure out how to include them in the formulas.
https://www.mediahuman.net/files/YouTubeDownloader-1012.dmg
https://www.mediahuman.net/files/YouTubeToMP3-1012.dmg

Comment (kvorobyov) [reply Casks/y/youtube-to-mp3.rb]: seems like i found how to do that

**Review by khipp — APPROVED** [review]
Thank you, @kvorobyov!

**Pull request merged**